RELEASE_NOTES.md

```diff
@@ -3,6 +3,19 @@ MSA Release Notes
 
 ### Next
 
+* Added additional detail and stack trace to error responses. This is enabled by default but can be
+disabled by setting `returnStackTraceInErrorResponse: false` in configuration file.
+* If multiple firstnames are provided by IDP, the MSA will select which to pass to local matching service based
```

Comment:
`If multiple firstnames are provided by an IDP, the MSA will select which one to pass to local matching service based on the following order`